sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestFramework/IssueLocationCollector.cs

```diff
@@ -208,29 +291,50 @@ private IEnumerable<string> GetIssueIds(Match match)
 
         private int GetOffset(Match match)
         {
-            var sign = match.Groups["sign"];
             var offset = match.Groups["offset"];
-            if (!sign.Success && !offset.Success)
+            return offset.Success ? int.Parse(offset.Value) : 0;
```

Comment:
`GetIssueIds` above: from `[abc, def, g.h.i]` it will return `abc, def, g` and strip the `.h.i` off. 
Why? Is it really needed? If yes, it should be documented in at the beginning of the file.

Reply:
Removing it did not affect anything apart a unit test relying on it, so I removed it as it does not seem necessary